apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -285,8 +301,12 @@ export default function RubricsContainer({
           text="Save your rubric"
           onClick={saveRubric}
           size={Button.ButtonSize.narrow}
+          disabled={saveNotificationText === 'Saving...'}
```

Comment:
I thought about adding state here for indicating when something should disabled, but since the notification text also provided information about if a user should be able to click the button again, I thought it made more sense just to reuse the `saveNotificationText` state here.

Reply:
don't want to belabor this but it might be more a little more resilient if you made constants for each of the save notification texts then compared to one of those.

So, something like:
```
const SAVING_TEXT = 'Saving...';
const SAVED_TEXT = 'Saved';
// etc

// a lot of JS here

<Button
  disabled={saveNotificationText === SAVING_TEXT}
/>

```

Otherwise, I think using `saveNotificationText` here works fine so no need for the extra state!

Reply:
This is actually perfect.  When making this change I kept thinking the `(saveNotificationText === 'Saving...')` is very breakable, but also didn't have a good sense of how to do it better - so thanks for pointing this out!  I made this change.